crates/tools/Cargo.toml

```diff
@@ -32,3 +33,5 @@ gw-common = { path = "../common" }
 gw-generator = { path = "../generator" }
 gw-jsonrpc-types = { path = "../jsonrpc-types" }
 url = "2.2"
+failure = "0.1.5"
```

Comment:
failure crate is abandoned, please use anyhow instead.

Reply:
Removed.